includes/block-patterns/course-list/class-sensei-course-list-block-patterns.php

```diff
@@ -43,43 +43,50 @@ public function register_course_list_block_patterns() {
 				'content'     => '<!-- wp:query {"query":{"offset":0,"postType":"course","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","perPage":6},"displayLayout":{"type":"list"},"layout":{"inherit":true}} -->
 					<div class="wp-block-query wp-block-sensei-lms-course-list wp-block-sensei-lms-course-list--is-list-view">
 						<!-- wp:post-template -->
-							<!-- wp:post-featured-image {"isLink":true,"align":"center"} /-->
-							<!-- wp:columns -->
-								<div class="wp-block-columns">
 
-									<!-- wp:column {"width":"66.66%"} -->
-										<div class="wp-block-column" style="flex-basis:66.66%">
-											<!-- wp:sensei-lms/course-categories /-->
+							<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"2px","right":"2px","bottom":"2px","left":"2px"}},"border":{"width":"1px","color":"#c7c3c34f"}},"layout":{"inherit":false}} -->
```

Comment:
I don't think 2px is quite enough space. 10px looks pretty good to me.

Reply:
Updated to 10px [here](https://github.com/Automattic/sensei/pull/5576/commits/bf2cdc1ffc508374fdac4c14abe13380e196c76d) and loved the outcome <3 Thanks!